src/components/ProjectPeople/ProjectPeople.tsx

```diff
@@ -2,40 +2,66 @@ import React from "react"
 import DeleteIcon from '@mui/icons-material/Delete';
 import EditIcon from '@mui/icons-material/Edit';
 import EmailIcon from '@mui/icons-material/Email';
+import PhoneIcon from '@mui/icons-material/Phone';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
 import './ProjectPeople.css'
 
 interface PersonDetails {
     name: string,
     involvement: string,
     description: string,
     email: string,
+    phone: string,
+    linkedIn: string,
+    isCurrentMember: boolean, // if person is current member or alumni
     edit?: () => void;
     delete?: () => void;
 }
 
 export const Person: React.FC<PersonDetails> = (props) => {
     return (
-        <div className="card">
+        <div className="person-card">
             <div className="card__header">
-                <h1 className="card__title">{ props.name }</h1>
+                <h1 className="card__title">{props.name}</h1>
                 <div className="card__header__icons">
-                    <EditIcon 
-                        className="icon" 
-                        onClick={props.edit}
-                    />
-                    <h3 className="icon icon--spacer">|</h3>
-                    <DeleteIcon
-                        className="icon" 
-                        onClick={props.delete}
-                    />
-                </div>
+                {(() => {
+                        if (props.isCurrentMember) {
+                            return (
+                                <span id="alumni-tag">Alumni</span>
+                            )
+                        }
+                })()}
+                {(() => {
+                        if (props.isCurrentMember) {
+                            return (
+                                <h3 className="icon icon--spacer">|</h3>
+                            )
+                        }
+                })()}
```

Comment:
Since both of these tags depend on the result of the same conditional statement, what do you think about combining them like so: 

```suggestion
if (props.isCurrentMember) {
                            return (
                                <>
                                <span id="alumni-tag">Alumni</span>
                                <h3 className="icon icon--spacer">|</h3>
                                </>
                            )
                        }
```